client/components/app-promo/test/promo-retriever.js

```diff
@@ -0,0 +1,17 @@
+import { expect } from 'chai';
```

Comment:
By convention, I'd have expected the tests for `app-promo/lib/promo-retriever.js` to live at `app-promo/lib/test/promo-retriever.js` (i.e. tests import one directory up).